test/src/graph/nodes/TransformPointsNodeTest.cpp

```diff
@@ -0,0 +1,89 @@
+#include <RGLFields.hpp>
+#include <graph/Node.hpp>
+#include <gtest/gtest.h>
+#include <repr.hpp>
+#include <utils.hpp>
+
+class TransformPointsNodeTest : public RGLAutoCleanupTestWithParam<std::tuple<int, rgl_mat3x4f>>, public RGLPointTestHelper {
+};
+
+INSTANTIATE_TEST_SUITE_P(
+    TransformPointsNodeTests, TransformPointsNodeTest,
+    testing::Combine(
+        testing::Values(1, 10, 100000),
+        testing::Values(identityTestTransform, translationTestTransform, rotationTestTransform, scalingTestTransform, complexTestTransform)));
+
+TEST_P(TransformPointsNodeTest, invalid_arguments)
+{
+    auto transform = std::get<1>(GetParam());
```

Comment:
Magic constant. IMHO, this could be easily eliminated by adding named indices (e.g. constexprs) together with `TransformPointsNodeTest` definition, and then you'd have:
```suggestion
    auto transform = std::get<TEST_PARAM_TRANSFORM_ID>(GetParam());
```

Reply:
Also, this `std::get<1>(GetParam())` is not necessary, because later lambda is called.

Reply:
Let's stick to:
` auto [_, transform] = GetParam();`